src/main/java/com/netflix/imflibrary/st2067_2/IMFCompositionPlaylistType.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *
+ * Copyright 2016 Netflix, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package com.netflix.imflibrary.st2067_2;
+
+import com.netflix.imflibrary.utils.UUIDHelper;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * A class that models an IMF Composition Playlist structure.
+ */
+@Immutable
+public final class IMFCompositionPlaylistType {
```

Comment:
@svenkatrav I am not sure if a class with fields that have "protected" access can be annotated as "Immutable"